app/Config/Logger.php

```diff
@@ -125,7 +125,7 @@ class Logger extends BaseConfig
             'path' => '',
         ],
 
-        /**
```

Comment:
Why are these happening?

Reply:
This is a hack actually on my part. Since the underlying code is commented out, php-cs-fixer treats the documentable structural element as the parent element.